MainWindow.xaml.cs

```diff
@@ -1,61 +1,39 @@
 ﻿using Newtonsoft.Json;
 using System;
 using System.Collections.Generic;
-using System.Linq;
 using System.Net.Http;
-using System.Text;
-using System.Threading.Tasks;
 using System.Windows;
-using System.Windows.Controls;
-using System.Windows.Data;
-using System.Windows.Documents;
-using System.Windows.Input;
-using System.Windows.Media;
-using System.Windows.Media.Imaging;
-using System.Windows.Navigation;
-using System.Windows.Shapes;
 
-namespace Program_WPF_API_z_NBP
+namespace Program_WPF_API_z_NBP;
+
+public partial class MainWindow : Window
 {
-   
-    public partial class MainWindow : Window
+    public MainWindow()
     {
-        public MainWindow()
-        {
-            InitializeComponent();
-            
-        }
-
-
-
-        private void button_Click(object sender, RoutedEventArgs e)
-        {
-            Program;
-
-            static async Task Main()
```

Comment:
Jak już robimy w WPF-ie to nie mamy funkcji Main. Jej odpowiednikiem jest klasa App, tam się ustawia kwestie startu aplikacji, ale w takim prostym przypadku nie trzeba tam nic dodawać